### ISSUE TITLE
as a user, i want to see who is lurking on my profile, so i can know who is interested in me, so i can build a professional relationship

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#3559]
### User Story

As a user, i want to see who is lurking on my profile, so i can know who is interested in me, so i can build a professional relationship.

### Why Is this Needed

We've received feedback from funders about the quality of developers, and finding the right talent to work on issues. We can extrapolate this kind of "seeking" behavior into the future, as both funders and contributors alike will hopefully use a user's Gitcoin profile as a statistical indication of user metrics they'd like to see that "signal" this kind of quality. 

Similar to LinkedIn, knowing who is interested in you can be quite powerful (and somewhat exciting too). It is similar almost to a warm introduction in your network, and from the Gitcoin side, can be setup as a premium feature.  

TLDR - seeing who is checking you out can drive relationships and be pitted as a premium feature for revenue.

### Current Behavior

No behavior like this exists.

### Expected Behavior

If you are a subscriber (past a paywall?) you can see a module that shows who has viewed your profile.

### Definition of Done

There needs to be design first on this ticket?

**Comment by PixelantDesign** [comment]
Cool. This seems like it could apply to both sides of the market.
